boards/arm/mimxrt1050_evk/Kconfig.defconfig

```diff
@@ -40,7 +40,7 @@ config GPIO_MCUX_IGPIO_5
 endif # GPIO_MCUX_IGPIO
 
 config I2C
-	default y if LVGL
+	default y if KSCAN
```

Comment:
What about `if (LVGL || KSCAN)` ?
We might want to enable LVGL w/o KSCAN. This is not what is done below in this file, but this is questionable.
@MaureenHelm, what is your opinion ?

Reply:
I'll do whatever you think is best on this one. Lower down in the file is...
```config KSCAN
	default y if LVGL
```
So there is some dependancy chain there.

Reply:
The dependency is for KSCAN; LVGL doesn't use I2C.

Reply:
So the question is if we have KSCAN as a dependancy of LVGL?